tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -19,8 +19,11 @@ def __init__(self, marionette):
         self.apps = GaiaApps(self.marionette)
         self.accessibility = Accessibility(self.marionette)
         self.frame = None
-        self.manifest_url = hasattr(self, 'manifest_url') and self.manifest_url or None
```

Comment:
Still wait for @JohanLorenzo review for this file to be sure, what I did is right or need to add something for that I comment it, but probably it will be removed we need to define manifest_url as null at the first that's I guess